guides/source/threading_and_code_execution.md

```diff
@@ -173,46 +318,54 @@ thread is mid-execution. Child threads should use the Executor instead.
 
 Active Job also wraps its job executions with the Reloader, loading the latest
 code to execute each job as it comes off the queue.
 
-Action Cable uses the Executor instead: because a Cable connection is linked to
-a specific instance of a class, it's not possible to reload for every arriving
-WebSocket message. Only the message handler is wrapped, though; a long-running
-Cable connection does not prevent a reload that's triggered by a new incoming
-request or job. Instead, Action Cable uses the Reloader's `before_class_unload`
-callback to disconnect all its connections. When the client automatically
-reconnects, it will be speaking to the new version of the code.
+Action Cable uses the Executor instead. A Cable connection is linked to a
+specific instance of a class, which means it's not possible to reload for every
+arriving WebSocket message. Only the message handler is wrapped, though; a
+long-running Cable connection does not prevent a reload that's triggered by a
+new incoming request or job. Instead, Action Cable also uses the Reloader's
+`before_class_unload` callback to disconnect all its connections. When the
+client automatically reconnects, it will be interacting with the new version of
+the code.
 
 The above are the entry points to the framework, so they are responsible for
 ensuring their respective threads are protected, and deciding whether a reload
-is necessary. Other components only need to use the Executor when they spawn
-additional threads.
+is necessary. Most other components only need to use the Executor when they
+spawn additional threads.
 
 ### Configuration
 
-The Reloader only checks for file changes when `config.enable_reloading` is
-`true` and so is `config.reload_classes_only_on_change`. These are the defaults in the
-`development` environment.
+#### Reloader and Executor Configuration
+
+The Reloader only checks for file changes when
+[`config.enable_reloading`](configuring.html#config-enable-reloading) and
+[`config.reload_classes_only_on_change`](configuring.html#config-reload-classes-only-on-change)
+are both `true`. These are the defaults in the `development` environment.
 
 When `config.enable_reloading` is `false` (in `production`, by default), the
 Reloader is only a pass-through to the Executor.
 
 The Executor always has important work to do, like database connection
 management. When `config.enable_reloading` is `false` and `config.eager_load` is
 `true` (`production` defaults), no reloading will occur, so it does not need the
-Load Interlock. With the default settings in the `development` environment, the
-Executor will use the Load Interlock to ensure constants are only loaded when it
-is safe.
+[Load Interlock](#load-interlock). With the default settings in the
+`development` environment, the Executor will use the Load Interlock to ensure
+constants are only loaded when it is safe.
+
+You can read more about the Load Interlock in the following section.
```

Comment:
If this line was not here, the same screen-position would be occupied by a "Load Interlock" heading; I feel like that conveys the same information?